lib/signaling/v2/peerconnectionmanager.js

```diff
@@ -135,8 +140,11 @@ PeerConnectionManager.prototype._getOrCreate = function _getOrCreate(id, configu
         self._closedPeerConnectionIds.add(peerConnection.id);
       }
     });
-    peerConnection.addMediaStream(this._localMediaStream);
+
     this._dataTrackSenders.forEach(peerConnection.addDataTrackSender, peerConnection);
+    this._mediaTrackSenders.forEach(function(mediaTrackSender) {
+      peerConnection.addMediaTrackSender(mediaTrackSender, this._localMediaStream);
```

Comment:
IMO, it would be nice if the APIs for `addDataTrackSender` and `addMediaTrackSender` could be made even more consistent by losing the MediaStream parameter. WDYT about passing `_localMediaStream` at construction time to the PeerConnectionV2? PeerConnectionV2 can then be responsible for using it when it calls `addTrack` internally.